pkg/util/validation/limits.go

```diff
@@ -62,6 +62,9 @@ type Limits struct {
 	CardinalityLimit    int           `yaml:"cardinality_limit"`
 	MaxCacheFreshness   time.Duration `yaml:"max_cache_freshness"`
 
+	// Ruler defaults and limits.
+	EvaluationDelay time.Duration `yaml:"evaluation_delay_duration"`
```

Comment:
Being the YAML structure generic for all Cortex services, I'm wondering if `evaluation_delay_duration` is too much generic and `ruler_evaluation_delay_duration` may be more appropriate.

Reply:
I agree